src/cdk-experimental/drag-drop/drag-drop-registry.ts

```diff
@@ -20,35 +18,35 @@ const activeEventOptions = supportsPassiveEventListeners() ? {passive: false} :
 type PointerEventHandler = (event: TouchEvent | MouseEvent) => void;
 
 /**
- * Service that keeps track of all the `CdkDrag` and `CdkDrop` instances, and
- * manages global event listeners on the `document`.
+ * Service that keeps track of all the drag item and drop container
+ * instances, and manages global event listeners on the `document`.
```

Comment:
It would be good to have a (non-public-facing) comment in this file somewhere that explains _why_ this is genericized instead of using concrete types.